test/unit/utils.test.js

```diff
@@ -3,13 +3,15 @@ const {
   eachAsync,
   executeLegacyOperation,
   makeInterruptibleAsyncInterval,
-  BufferPool
+  BufferPool,
+  shuffle
 } = require('../../src/utils');
 const { expect } = require('chai');
 const sinon = require('sinon');
+const { MongoRuntimeError } = require('../../src/error');
 
-describe('utils', function () {
-  context('eachAsync', function () {
+describe('driver utils', function () {
+  context('eachAsync function', function () {
```

Comment:
purely preference, but have you considered using parentheses to indicate that a method is being tested? e.g., `describe('eachAsync()')`

Reply:
It's maybe a bad reason but if we don't name our tests something different than the variable that gets imported at the top, the explorer puts the run/debug buttons up at the require, instead of above the test. So I'm up for any naming, its just a small QoL.